### ISSUE TITLE
NoSuchMethod issue on init

### ISSUE THREAD

**Issue by aematsubara** [CryptoMorin/XSeries#226]
**Description**
I got this issue in the latest minecraft version, which happens when I use ReflectionUtils.MINOR_NUMBER to check the server version (tho I use this utility for other stuff too).

**Version**
Spigot and paper 1.20.2 (tested with both softwares), using XSeries 9.5.0

```
[16:18:17] [Server thread/WARN]: java.lang.NoSuchMethodException: no such method: net.minecraft.server.network.PlayerConnection.a(Packet)void/invokeVirtual
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:976)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1117)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:3649)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.findVirtual(MethodHandles.java:2680)
[16:18:17] [Server thread/WARN]: 	at me.matsubara.roulette.util.xseries.ReflectionUtils.<clinit>(ReflectionUtils.java:240)
[16:18:17] [Server thread/WARN]: 	at me.matsubara.roulette.RoulettePlugin.onEnable(RoulettePlugin.java:76)
[16:18:17] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266)
[16:18:17] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342)
[16:18:17] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480)
[16:18:17] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugin(CraftServer.java:548)
[16:18:17] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugins(CraftServer.java:462)
[16:18:17] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:577)
[16:18:17] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:400)
[16:18:17] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:250)
[16:18:17] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:954)
[16:18:17] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:298)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[16:18:17] [Server thread/WARN]: Caused by: java.lang.NoSuchMethodError: 'void net.minecraft.server.network.PlayerConnection.a(net.minecraft.network.protocol.Packet)'
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:1085)
[16:18:17] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1114)
[16:18:17] [Server thread/WARN]: 	... 15 more
```

**Comment by BySwiizen** [comment]
The plugin does not support 1.20.2 at the moment I think.

**Comment by CryptoMorin** [comment]
Fixed in v9.6.0
